public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -559,21 +559,24 @@ export class LokiDatasource extends DataSourceApi<LokiQuery, LokiOptions> {
           throw err;
         }),
         switchMap((response: { data: { values: string[] }; status: number }) =>
-          iif<DataQueryResponse, DataQueryResponse>(
+          iif<DataQueryResponse, any>(
```

Comment:
Had to change this because the object was a real browser response object, as opposed to a `DataQueryResponse`.